db/migrate/20231215201714_create_messages.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMessages < ActiveRecord::Migration[7.1]
+  def change
+    create_table :messages do |t|
+      t.string :content
```

Comment:
- You have done great so far :+1: :+1: But there are a few requirements you still need to meet before moving to the next milestone. :+1: :100: 
- [ ] You did great generating the `Migrations` as suggested by the previous reviewer, but you did not generate the `Message Model`. Models are responsible for querying and interacting with the database and therefore you need to have the Message model` so that you can add greetings to the database and also fetch random greetings from the database. :wave: :smile:  You can run the cmd below to generate the model.
```ruby
rails g model Message --skip
````